docker/sonar-scanner.yml

```
@@ -0,0 +1,27 @@
+      -Dsonar.java.test.binaries=./target/test-classes
+      -Dsonar.java.libraries=./.m2/repository/**/*.jar
+      -Dsonar.coverage.jacoco.xmlReportPaths=./target/site/jacoco/jacoco.xml
+      -Dsonar.scm.provider=git
+      -Dsonar.host.url=http://sonarqube:9000
+      -Dsonar.token=${SONAR_TOKEN}
+    networks:
+      - monitoring-network
+
+networks:
+  monitoring-network:
+    driver: bridge
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)